extensions/dub-link-shortener/src/search-links.tsx

```diff
@@ -20,14 +20,17 @@ import { MutatePromise, showFailureToast } from "@raycast/utils";
 import { LinkSchema } from "@/types";
 
 export default function SearchLinks() {
-  const [workspaceId, setWorkspaceId] = useState<string>("");
+  const [workspaceId, setWorkspaceId] = useState<string | undefined>(undefined);
   const { workspaces, isLoading: isLoadingWorkspaces, error: workspacesError } = useWorkspaces();
+
+  const isOnlyWorkspace = !isLoadingWorkspaces && !workspacesError && workspaces && workspaces.length === 1;
```

Comment:
since this is the same derived state in many places, thoughts on returning this as part of useWorkspaces()?

Reply:
Good call, also we don't need `error` in this condition and need to specify it separately.